rs/sns/governance/proto/ic_sns_governance/pb/v1/governance.proto

```diff
@@ -68,6 +68,12 @@ message NeuronId {
   bytes id = 1;
 }
 
+// A sequence of NeuronIds, which is used to get prost to generate a type isomorphic to Option<Vec<NeuronId>>.
+message NeuronIds {
```

Comment:
I don't think this is a good way to do it – if you asked me "Where does NeuronIds" live, I'd tell you "Probably by NeuronId". I wouldn't say "By ClaimSwapNeuronRequest since that's the only place that uses NeuronIds".

Reply:
That's fine, I didn't realize `message NeuronId` is defined right above.